src/components/navigation/header/config/drawerNodeItems.ts

```diff
@@ -204,7 +242,7 @@ export const drawerNodeItems = ({
             de: '/de/besuchte-fahrzeuge',
             en: '/de/besuchte-fahrzeuge',
             fr: '/fr/vehicules-visites',
-            it: '/it/veicoli-visitat',
+            it: '/it/veicoli-visitati',
```

Comment:
We remove [this](https://github.com/as24ch/www/pull/2164/files#diff-a7589bd967e111f1465826fb0d8e46c361dd2239cc8434f4230d619fba6e4e23L1) for the PDP release. You could drop it, should be merged today

Reply:
Ok, so I should remove this link item from navigation?

Reply:
yes please :)